src/main/webapp/script.js

```diff
@@ -1,13 +1,10 @@
-function getRandomRestaurant() {
-  const restaurants =
-      ['Din Tai Fung', 'Mcdonalds', 'KFC', 'Subway'];
-
+function getRandomRestaurant(restaurants) {
   // Pick a random restaurant.
   const restaurant = restaurants[Math.floor(Math.random() * restaurants.length)];
 
   // Add it to the page.
   const resElement = document.getElementById('restaurant-name');
-  resElement.innerText = restaurant;
+  resElement.innerHTML = "<p>" + restaurant.name + "</p><p>" + JSON.stringify(restaurant.geometry.location) + "</p>";
```

Comment:
The id tag of the element won't reflect what is actually being written inside.

We can create a new empty HTML element, with the id- "restaurant-loc" and write the location there, or instead, we can change the ID of the p tag in HTML to 'restaurant'.